ARTICLE-2.markdown

```diff
@@ -2,8 +2,8 @@
 
 ### Section 2.1  Voluntary Withdrawal
 
-A Member shall have the right to resign from the Corporation, effective
-immediately without any action on the part of the Corporation.
+A Member shall have the right to resign from the Cooperative, effective
+immediately without any action on the part of the Cooperative.
 
```

Comment:
Most cooperatives do not agree. If a member resigns, the initial capital contribution and patronage should not be withheld by the cooperative if the member no longer benefits from it.